[1.10] cgroup panic

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0xb1a82f]

goroutine 1 [running]:
github.com/opencontainers/runc/libcontainer/cgroups/fs.(*Manager).Freeze(0xc820c418e0, 0x16cdfd8, 0x6, 0x0, 0x0)
	/go/src/github.com/docker/docker/vendor/src/github.com/opencontainers/runc/libcontainer/cgroups/fs/apply_raw.go:205 +0x2bf
github.com/opencontainers/runc/libcontainer.killCgroupProcesses(0x7f5fe456fc30, 0xc820c418e0, 0x0, 0x0)
	/go/src/github.com/docker/docker/vendor/src/github.com/opencontainers/runc/libcontainer/init_linux.go:309 +0x8c
github.com/opencontainers/runc/libcontainer.(*linuxContainer).Destroy(0xc820d88ea0, 0x0, 0x0)
	/go/src/github.com/docker/docker/vendor/src/github.com/opencontainers/runc/libcontainer/container_linux.go:331 +0x238
github.com/docker/docker/daemon/execdriver/native.(*Driver).Terminate(0xc820717840, 0xc8207e5c00, 0x0, 0x0)
	/go/src/github.com/docker/docker/daemon/execdriver/native/driver.go:346 +0x2dd
github.com/docker/docker/daemon.(*Daemon).Register(0xc82000c840, 0xc820bfcfc0, 0x0, 0x0)
	/go/src/github.com/docker/docker/daemon/daemon.go:276 +0x4df
github.com/docker/docker/daemon.(*Daemon).restore(0xc82000c840, 0x0, 0x0)
	/go/src/github.com/docker/docker/daemon/daemon.go:343 +0x11c5
github.com/docker/docker/daemon.NewDaemon(0xc820134c80, 0xc820028118, 0x0, 0x0, 0x0)
	/go/src/github.com/docker/docker/daemon/daemon.go:835 +0x335f
main.(*DaemonCli).CmdDaemon(0xc820648a30, 0xc82000a070, 0x3, 0x3, 0x0, 0x0)
	/go/src/github.com/docker/docker/docker/daemon.go:227 +0x136a
reflect.callMethod(0xc820646d80, 0xc820d43c68)
	/usr/local/go/src/reflect/value.go:628 +0x1fc
reflect.methodValueCall(0xc82000a070, 0x3, 0x3, 0x1, 0xc820646d80, 0x0, 0x0, 0xc820646d80, 0x0, 0x473375, ...)
	/usr/local/go/src/reflect/asm_amd64.s:29 +0x36
github.com/docker/docker/cli.(*Cli).Run(0xc820646cf0, 0xc82000a060, 0x4, 0x4, 0x0, 0x0)
	/go/src/github.com/docker/docker/cli/cli.go:89 +0x383
main.main()
	/go/src/github.com/docker/docker/docker/docker.go:63 +0x43c
```

I had just upgraded a 1.9.1 environment to 1.10 and this panic was thrown on the first daemon start.  This is running off of 63525a6dc5f7be370736ec2905d40a8c8fdf1162

--- a/hack/vendor.sh
+++ b/hack/vendor.sh
@@ -57,7 +57,7 @@ clone git github.com/miekg/pkcs11 80f102b5cac759de406949c47f0928b99bd64cdf
 clone git github.com/jfrazelle/go v1.5.1-1
 clone git github.com/agl/ed25519 d2b94fd789ea21d12fac1a4443dd3a3f79cda72c
 
-clone git github.com/opencontainers/runc d97d5e8b007e4657316eed76ea30bc0f690230cf # libcontainer
+clone git github.com/opencontainers/runc 47e3f834d73e76bc2a6a585b48d2a93325b34979 # libcontainer
 clone git github.com/seccomp/libseccomp-golang 1b506fc7c24eec5a3693cdcbed40d9c226cfc6a1
 # libcontainer deps (see src/github.com/opencontainers/runc/Godeps/Godeps.json)
 clone git github.com/coreos/go-systemd v4

